test/torchaudio_unittest/functional/functional_impl.py

```diff
@@ -106,6 +106,84 @@ def test_lfilter_9th_order_filter_stability(self):
         yhat = F.lfilter(x, a, b, False)
         self.assertEqual(yhat, y, atol=1e-4, rtol=1e-5)
 
+    def test_filtfilt_simple(self):
+        """
+        Check that, for an arbitrary signal, applying filtfilt with filter coefficients
+        corresponding to a pure delay filter imparts no time delay.
+        """
+
+        torch.random.manual_seed(42)
+        waveform = torch.rand(2, 44100 * 1, dtype=self.dtype, device=self.device)
```

Comment:
Can you use `get_whitenoise` or `get_sinusoid`?
Also can you use smaller sample rate like 8000 / 16000? (<- this has nothing to do with the quality of the test but we often see timeout in fbcode)

Reply:
sure, done